### DISCUSSION TITLE
Card ~grid with Pagination example / recommendation?

### DISCUSSION THREAD

**Discussion by StephenOTT** [microsoft/fluentui-blazor#1390] [Q&A]
Hi

Is there any examples or recommendations for building FluentCard lists.  Essentially all the same features of DataGrid but with a Card view so you can show multiple cards on a single row.

thanks

**Comment by eramrit78** [comment]
Me to

**Comment by vnbaaij** [comment]
I think you could use the FluentGrid (**not** FluentDataGrid) for that. Except for the pagination part. Not sure how you see that

**Reply by eramrit78** [reply]
A pagination component required. just like https://demos.blazorbootstrap.com/pagination
